backend/app/views/spree/admin/orders/_store_form.html.erb

```diff
@@ -1,18 +1,30 @@
-<div data-hook="store-form-container">
-  <%= form_for [:admin, order], method: :put, url: set_store_admin_order_url do |f| %>
-    <fieldset>
-    <div data-hook="admin_order_store_field">
-      <%= f.field_container :store_id, class: ['form-group'] do %>
-        <%= f.label :store_id, Spree.t(:store) %>
-        <%= f.collection_select(:store_id, @stores, :id, :unique_name, { include_blank: false }, { class: "select2" }) %>
-        <%= f.error_message_on :store_id %>
-      <% end %>
+<div data-hook="store-form-container col-12">
```

Comment:
The changes in this file are with the intention that once an order is marked as completed, you should not be able to change the store the order belongs to.

Mainly because products are scoped by store, so when we change the store_id on the order we now call `empty!` on the order creating an empty cart as not to risk dragging products through that are not actually available to the store that the order is now belongs to since updating.